### DISCUSSION TITLE
Templates to add to Marketplace

### DISCUSSION THREAD

**Discussion by Lory1990** [mia-platform/community#119] [Ideas]
Hi,
I am Lorenzo from Step4Business, during the last 6 months I and my team developed some templates for Mia platform

* __Quarkus__: https://github.com/Lory1990/mia-platform-quarkus-template/
* __Quarkus Native__: https://github.com/Lory1990/mia-platform-quarkus-native-template/
* __Gatsby__: https://github.com/Lory1990/mia-platform-gatsby-template/
* __NextJS__: https://github.com/Lory1990/mia-platform-nextjs-template/
* __PHP__: https://github.com/Lory1990/mia-platform-php-template/

Please add it to the Marketplace 👍

**Comment by fredmaggiowski** [comment]
Hi @Lory1990 thank you for reaching out, we can add them for sure! @albertotessarotto and I are going to contact you as soon as possible to ask you a few details and add them properly.

**Reply by FabioNappi** [reply]
Hi guys!

Any news about these new templates?

**Reply by Lory1990** [reply]
Hi Fabio!
me, @fredmaggiowski are working on it 🎉😁

Fede do you have any news?

**Reply by fredmaggiowski** [reply]
Actually @FedericoOldrini is in charge of uploading them in the marketplace I think he has planned it but I don't know the timeline he has in mind.

**Comment by Lory1990** [comment]
Hello everyone,
We have left this activity behind. When are we putting these templates on the marketplace?

**Reply by fredmaggiowski** [reply]
Hey, yes we definitely did, sorry for the delay. I've reached out to @FedericoOldrini that will let you know asap
